chef/cookbooks/nova_dashboard/recipes/server.rb

```diff
@@ -297,8 +277,7 @@
     :memcached_locations => memcached_locations,
     :can_set_mount_point => node["nova_dashboard"]["can_set_mount_point"],
     :can_set_password => node["nova_dashboard"]["can_set_password"],
-    :keystone_api_version => keystone_api_version,
-    :keystone_internal_auth_url => keystone_internal_auth_url,
+    :multi_domain_support => node["nova_dashboard"]["multi_domain_support"],
```

Comment:
```validate_proposal_after_save``` doesn't allow to enable multi domain support, when keystone API version is set to 2.0. Do you think that's not enough?